index.js

```diff
@@ -11,6 +11,8 @@ const Vec3 = require('vec3').Vec3
 const Physics = require('./lib/physics')
 const nbt = require('prismarine-nbt')
 
+const passableEnts = require('./lib/passableEnts.json')
```

Comment:
Is there a reason to make this const? I see some off the entities are things like fireballs or ender dragon balls. Some plugins or user code could make pathfinder avoid those too if the list is editable.

Reply:
Implemented. I made the `passableEnts` as well as `interactableBlocks` into public fields on Movements. I also added another field `entsToAvoid` which will cause the indexer to add the max cost to any blocks containing those entities. I figure these could be useful for anyone trying to add their own legacy/modded additions to custom movement types.